types/react-native-scrollable-tab-view/react-native-scrollable-tab-view-tests.tsx

```diff
@@ -3,6 +3,7 @@ import { Text, TextStyle, View, ViewStyle } from 'react-native';
 import ScrollableTabView, { ScrollableTabBar, TabProps, DefaultTabBar } from 'react-native-scrollable-tab-view';
 
 interface MyTextProps {
+    children?: React.ReactNode;
```

Comment:
What a strange api? Probably best to expose a helper type like `children?: React.Children`.